target/linux/mediatek/dts/mt7986a-asus-rt-ax59u.dts

```diff
@@ -230,17 +232,42 @@
 			partition@400000 {
 				label = "UBI_DEV";
 				reg = <0x400000 0x7c00000>;
+				compatible = "linux,ubi";
```

Comment:
`compatible` should be first, see https://www.kernel.org/doc/html/next/devicetree/bindings/dts-coding-style.html#order-of-properties-in-device-node